main.go

```diff
@@ -124,9 +124,7 @@ func getFromCache(word string) []prompt.Suggest {
 	completer, found := memoryCache.Get(cacheKey)
 	if !found {
 		completer = imageFetchCompleter(word, 10)
-		if len(completer.([]prompt.Suggest)) > 0 {
-			memoryCache.Set(cacheKey, completer, cache.DefaultExpiration)
-		}
+		memoryCache.Set(cacheKey, completer, cache.DefaultExpiration)
```

Comment:
In this case if we faced with timeout, it will cache empty for even valid images. Do we really want to cache it?

Reply:
If there's no response from context or Hub API, caching will be disabled.